recipes/libbsd/all/conanfile.py

```diff
@@ -32,51 +36,70 @@ def build_requirements(self):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def export_sources(self):
+        export_conandata_patches(self)
+
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        try:
+            del self.settings.compiler.libcxx
+        except ConanException:
+            pass
+        try:
+            del self.settings.compiler.cppstd
+        except ConanException:
+            pass
+    
+    def generate(self):
+        if not cross_building(self):
+            env = VirtualRunEnv(self)
+            env.generate(scope="build")
+        tc = AutotoolsToolchain(self)
+        if is_apple_os(self):
```

Comment:
redundant if MacOS is not supported

Reply:
😕 https://github.com/conan-io/conan-center-index/blob/0cfba12866ccf53adfe6eb6fcc228f397801aba6/recipes/libbsd/all/conandata.yml#L7 

it looks like support is being patched in?

Reply:
your are right, I missed that. Let me take a look...